client/dashboard/components/page-layout/index.tsx

```diff
@@ -21,15 +21,18 @@ function PageLayout( {
 	actions,
 	children,
 	size = 'large',
+	callOut,
 }: {
 	title: string;
 	description?: React.ReactNode;
 	actions?: React.ReactNode;
 	children?: React.ReactNode;
 	size?: 'large' | 'small';
+	callOut?: React.ReactNode;
 } ) {
 	return (
 		<VStack spacing={ 8 } className="dashboard-page-layout" style={ sizes[ size ] }>
+			{ callOut && <CallOutParent>{ callOut }</CallOutParent> }
```

Comment:
I've added the call out directly to the `PageLayout`. I'd prefer to experiment with call outs before wrapping them into any sort of layout. However it needs to cover the heading, and this is where the heading gets rendered.